src/backupNg/backupNg.spec.js

```diff
@@ -11,6 +12,84 @@ const DEFAULT_SCHEDULE = {
   cron: "0 * * * * *",
 };
 
+const validateRootTask = (log, props) =>
+  expect(log).toMatchSnapshot({
+    end: expect.any(Number),
+    id: expect.any(String),
+    jobId: expect.any(String),
+    scheduleId: expect.any(String),
+    start: expect.any(Number),
+    ...props,
+  });
+
+const validateVmTask = (task, vmId, props = {}) => {
+  if (task.status !== "success") {
```

Comment:
I think it should be passed by the test. In case if the error is not propagated.